src/helpers/blockTime.js

```diff
@@ -0,0 +1,29 @@
+const MILLISECONDS_IN_A_SECOND = 1000
+
+export const blockTime = (eth) =>
+  async (blockNumber, showBlock = true, averageBlockTime = 13.965) => {
```

Comment:
```suggestion
  async (blockNumber, showBlock = true, averageBlockTime = 15) => {
```

Given the ice age and hardfork shenanigans, it's pretty hard to come up with a value for this so I'd suggest either expecting 10, 12, or 15s blocks (I forget exactly what the average target is).